rusk/src/bin/config/http.rs

```diff
@@ -179,12 +214,23 @@ mod tests {
     #[test]
     fn deserialize_config() {
         let config_str = r#"listen = true
-                            feeder_call_gas = 18446744
+                            feeder_call_gas = "18446744"
                             ws_sub_channel_cap = 16
                             ws_event_channel_cap = 1024
                             headers = [["name1", "value1"], ["name2", "value2"]]"#;
 
         toml::from_str::<HttpConfig>(config_str)
             .expect("deserializing config should succeed");
     }
+
+    #[test]
+    fn deserialize_invalid_feeder_call_gas() {
+        let config_str = r#"feeder_call_gas = "invalid_number""#;
+        let result = toml::from_str::<HttpConfig>(config_str);
+        assert!(result.is_err());
+
+        let config_str = r#"feeder_call_gas = 18446744"#;
+        let result = toml::from_str::<HttpConfig>(config_str);
+        assert!(result.is_err());
```

Comment:
since the parsing is broken for values larger than `i64::MAX` it would be nice to also have a test-case that covers such a number